contents/docs/self-host/deploy/configuration.md

```diff
@@ -89,39 +96,56 @@ postgresql:
   persistence:
     size: 60Gi
 
-redis:
-  master:
-    size: 30Gi
-
 kafka:
   persistence:
     size: 30Gi
   logRetentionBytes: _22_000_000_000
 
-# Enable horizontal autoscaling for services
-pgbouncer:
+# Enable horizontal pod autoscaling for stateless services
+events:
   hpa:
     enabled: true
 
-web:
+pgbouncer:
   hpa:
     enabled: true
 
-beat:
+plugins:
   hpa:
     enabled: true
 
-worker:
+web:
   hpa:
     enabled: true
 
-plugins:
+worker:
   hpa:
     enabled: true
+
 ```
 
 </details>
 
+For the stateful services (ClickHouse, Kafka, Redis, PostgreSQL, Zookeeper), we suggest you to run them on nodes with dedicated CPU resources and fast drives (SSD/NVMe).
+
+In order to do so, after having labeled your Kubernetes nodes, you can assign pods to them using the following overrides:
```

Comment:
What about our services? events/web/plugins etc?

Reply:
I don't think that should be needed.

Reply:
but I can document how to do that

Reply:
Agreed it shouldn't be needed, but stonly specifically asked for it - I guess to avoid extra CPU load on the clickhouse nodes from other pods?

Reply:
As we currently do not expose `nodeSelector`, `affinity` and `tolerations` options to all our stateless components I'm inclined to merge this for now and add additional information about ☝️ once the support will land.